tests/format/x/html/html-void-tags/jsfmt.spec.js

```diff
@@ -1,4 +1,15 @@
-// using bogus option to keep snapshot more consistent with prettierX 0.18.x
-run_spec(__dirname, ["html", "angular"], { bogus: null });
+run_spec(__dirname, ["html", "angular", "vue"], {
+  errors: {
+    vue: ["input-void-element.html"], // [TBD] no error with `vue` in prettierX 0.18.x
```

Comment:
This error with `vue` came with Prettier 2.1.0 update.

**Playground on <https://github.com/prettier/prettier/pull/8127> soon after Prettier 2.0.5 release:**

**Prettier pr-8127**
[Playground link](https://deploy-preview-8127--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEAeAllADgVxgAhgE8s4BeAcngA8YKA+AHShABoQIsZ1oBnZUAEMATsIgB3AAoiE-FIIA24wUX7sARsMFgA1nBgBlLNswBzZDGE447OAFt1cACZPnAGUFRTOQabgAxCGE7QRhuL2QQQTwINhAACxg7BQB1ePR4XmMwOANZDPQANwyiSLBeNRBMXjhhGEktUxDkADNFGvYAK15qACEtXX0DQTs4N0w4VvabEG7qAzMFOABFHAh4KYUOkGNhGuFIwus4rGFMGBT0Jxh45AAOAAZ2U4galK0sSNO4fcLJ9gAjmt4A1OHIorwALRQODOZxxYRwIHoRENXzNJBtLYzGp2dAWKw4xYrYGTTHTdgwQTqS7XW5IABMlK06AUZgAwhA7BiQD8AKxxHA1AAq1LkWO2RzgAEkoK5YAYwGcuABBOUGYhLTY1AC+OqAA)
```sh
--parser vue
```

**Input:**
```vue
<input type='text'>

```

**Output:**
```vue
<input type="text" />

```

**Playground on <https://github.com/prettier/prettier/pull/9035> right before Prettier 2.1.0 release:**

**Prettier pr-9035**
[Playground link](https://deploy-preview-9035--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEAeAllADgVxgAhgE8s4BeAcngA8YKA+AHShABoQIsZ1oBnZUAEMATsIgB3AAoiE-FIIA24wUX7sARsMFgA1nBgBlLNswBzZDGE447OAFt1cACZPnAGUFRTOQabgAxCGE7QRhuL2QQQTwINhAACxg7BQB1ePR4XmMwOANZDPQANwyiSLBeNRBMXjhhGEktUxDkADNFGvYAK15qACEtXX0DQTs4N0w4VvabEG7qAzMFOABFHAh4KYUOkGNhGuFIwus4rGFMGBT0Jxh45AAOAAZ2U4galK0sSNO4fcLJ9gAjmt4A1OHIorwALRQODOZxxYRwIHoRENXzNJBtLYzGp2dAWKw4xYrYGTTHTdgwQTqS7XW5IABMlK06AUZgAwhA7BiQD8AKxxHA1AAq1LkWO2RzgAEkoK5YAYwGcuABBOUGYhLTY1AC+OqAA)
```sh
--parser vue
```

**Input:**
```vue
<input type='text'>

```

**Output:**
```vue
SyntaxError: Unexpected character "EOF" (2:1)
  1 | <input type='text'>
> 2 | 
    | ^
```

I think it was better in Prettier 2.05, which would actually fix the Vue code.

TODO:

- [ ] raise an issue to correct this, ideally in the upstream Prettier codebase